test/test_images/autoscale/autoscale.go

```diff
@@ -130,13 +147,20 @@ func handler(w http.ResponseWriter, r *http.Request) {
 	// Consume time, cpu and memory in parallel.
 	var wg sync.WaitGroup
 	defer wg.Wait()
-	if hasMs && ms > 0 {
+	if hasMs && !hasMssd && ms > 0 {
```

Comment:
In theory you can get away with just passing in `0` as mean and use generic text.
But fine by me.

Reply:
This whole thing could use some refactoring TLC, so I'm not going to split hairs.